app/queries.py

```diff
@@ -447,3 +447,48 @@ def get_queue_graph_data(center_id):
             break
 
     return fronta
+
+
+def get_vaccination_graph_data(center_id):
+    rezervace = pd.read_sql_query(
+        """
+        select datum, sum(maximalni_kapacita) kapacita, sum(maximalni_kapacita-volna_kapacita) rezervace,
```

Comment:
Please, use a filter -> data older than today -> otherwise we can see capacities on places and it can be misused